tools/alpine/Makefile

```diff
@@ -36,9 +53,9 @@ versions.$(ARCH): Makefile hash iid
 	docker run --rm $(shell cat iid) find /mirror -name '*.apk' -exec basename '{}' .apk \; | sort | (echo '# automatically generated list of installed packages'; cat -) >> versions.$(ARCH)
 
 push: hash iid versions.$(ARCH)
-	DOCKER_CONTENT_TRUST=$(DOCKER_CONTENT_VAR) docker pull $(ORG)/$(IMAGE):$(shell cat hash) || \
+	docker pull $(ORG)/$(IMAGE):$(shell cat hash) || \
```

Comment:
this is supposed to be a trusted pull if you are going to push with trust, as if there is an unsigned copy it will ignore it and replace it with your signed one.